src/applications/pensions/config/chapters/06-additional-information/helper.js

```diff
@@ -0,0 +1,4 @@
+import get from 'platform/forms-system/src/js/utilities/data/get';
```

Comment:
I think this helper.js is in theory a good idea even if this particular example isn't a strong case.